### DISCUSSION TITLE
Speech to text "isFinal" field always true

### DISCUSSION THREAD

**Discussion by Grzetan** [deepgram/community#688] [General help]
### Which Deepgram product feature are you using?

Deepgram API - STT Streaming

### Details

Hello,
I'm having a problem where the field `isFinal` in responses in always `true`. STT works fine and I get the transcribed text but I need to know when does it end.
When I print responses from deepgram I get:

![image](https://github.com/deepgram/community/assets/52281852/453635c4-dadc-4621-a943-86064a11493d)

As you can see, each response has both flags `isFinal` and `speech_final` set to `true`

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jjmaldonis** [comment]
Hi @Grzetan, you're looking for interim results. To enable these, add `interim_results=true` to your URL.

i.e. change `.../v1/listen?language=pl&model=nova-2&smart_format=true` to `.../v1/listen?language=pl&model=nova-2&smart_format=true&interim_results=true`.
